src/5e-SRD-Proficiencies.json

```diff
@@ -862,22 +1319,37 @@
 		"name": "Rogue"
 	}],
 	"races": [],
-	"url": "/api/proficiencies/thieves-tools"
-}, {
+	"url": "/api/proficiencies/thieves-tools",
+	"references": [{
+		"url": "/api/equipment/thieves-tools",
+		"type": "equipment",
+		"name": "Thieves' Tools"
+	}]
+  }, {
 	"index": "land-vehicles",
 	"type": "Vehicles",
 	"name": "Land Vehicles",
 	"classes": [],
 	"races": [],
-	"url": "/api/proficiencies/land-vehicles"
-}, {
+	"url": "/api/proficiencies/land-vehicles",
+	"references": [{
+		"url": "/api/equipment-categories/tack-harness-and-drawn-vehicles",
```

Comment:
I'm not too sure about this one, since it include saddles, barding, and bit and briddle. However, this appears to be the best option.

Reply:
I think that makes the most sense, unless you want to add a land vehicles Equipment Category, which I think would be completely reasonable in this case. Especially since we have a `Waterborne vehicles` category.

Reply:
I think I will make a category